### ISSUE TITLE
Unable to `build` Vue + Headless UI

### ISSUE THREAD

**Issue by dvlden** [wxt-dev/wxt#246]
### Describe the bug

It throws and fails to build the output.

### To Reproduce

[Reproduction.zip](https://github.com/wxt-dev/wxt/files/13491567/Archive.zip)

1. Install WXT with Vue
2. Install Headless UI `pnpm install @headlessui/vue`
3. Use one of the Headless UI components in the project _(in the Popup for example)_
4. Extend `wxt.config.ts` with:
```ts
imports: {
    presets: ['vue'],
    addons: {
      vueTemplate: true,
    },
  },
```
5. Run WXT's build script `pnpm run build`

### Expected behavior

Successfully compiles the project without conflicts. Maybe there's a way to also [auto-import](https://github.com/unplugin/unplugin-auto-import) Headless UI, but I haven't found it suggested by TS in the `imports.presets` array.

What resolves it for me is to configure `imports: false` and then manually do all of the importing where necessary.

### Screenshots

<img width="1192" alt="Screenshot 2023-11-27 at 18 24 12" src="https://github.com/wxt-dev/wxt/assets/3460368/53d1879a-7f20-4618-a5c0-7c4ccc06b1ef">

### Environment

```
  System:
    OS: macOS 14.1.1
    CPU: (12) arm64 Apple M2 Max
    Memory: 59.28 GB / 96.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.3.1 - ~/.nvm/versions/node/v20.3.1/bin/node
    Yarn: 1.22.21 - ~/.nvm/versions/node/v20.3.1/bin/yarn
    npm: 10.1.0 - ~/.nvm/versions/node/v20.3.1/bin/npm
    pnpm: 8.6.12 - ~/.nvm/versions/node/v20.3.1/bin/pnpm
    bun: 1.0.10 - /opt/homebrew/bin/bun
  Browsers:
    Chrome: 119.0.6045.159
    Safari: 17.1
  npmPackages:
    wxt: ^0.10.2 => 0.10.2 
```

**Comment by aklinker1** [comment]
Appears to be an issue with `unimport`, there's a similar problem for nuxt: https://github.com/nuxt/nuxt/issues/18823#issuecomment-1419704343

WXT doesn't provide the `imports.transform.exclude` option, but this is fixed by not adding imports to dependencies inside `node_modules`.

**Comment by aklinker1** [comment]
Released in [`v0.10.3`](https://github.com/wxt-dev/wxt/releases/tag/v0.10.3). Let me know if it doesn't work for you @dvlden
